### PR TITLE
Fix/1627  fix flake8 issues

### PR DESCRIPTION
**Pull request by inkhey**
related to #1627

### CODE REVIEW

File: backend/tracim_backend/lib/core/content.py [hunk 1]
```diff
@@ -1193,24 +1197,24 @@ def get_all_without_exception(
...
+    #         RevisionReadStatus.user_id == self._user_id
+    #     )
+    #
+    #     not_read_revisions = (
+    #         self._revisions_base_query(workspace)
+    #         .filter(~ContentRevisionRO.revision_id.in_(read_revision_ids))
+    #         .filter(ContentRevisionRO.workspace_id == Workspace.workspace_id)
+    #         .filter(Workspace.is_deleted.is_(False))
+    #         .subquery()
+    #     )
```
Comment (buxx) [inline backend/tracim_backend/lib/core/content.py]: why this code is commented ? Delete it if no longueur required.

File: backend/tracim_backend/lib/core/user.py [hunk 2]
```diff
@@ -233,7 +232,7 @@ def user_with_email_exists(self, email: str) -> bool:
             self.get_one_by_email(email)
             return True
         # TODO - G.M - 09-04-2018 - Better exception
```
Comment (buxx) [inline backend/tracim_backend/lib/core/user.py]: Can you open a technical debt issue for this ?

File: backend/tracim_backend/lib/webdav/authentification.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def isRealmUser(
         try:
             api.get_one_by_email(username)
             return True
-        except:
+        except Exception:
```
Comment (buxx) [inline backend/tracim_backend/lib/webdav/authentification.py]: Bad practice here: open an technical debt issue please.

File: backend/tracim_backend/lib/core/content.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: unused, dropped.

File: backend/tracim_backend/lib/core/user.py (hunk 2 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/user.py]: done #1635

File: backend/tracim_backend/lib/webdav/authentification.py (hunk 3 above)
Comment (inkhey) [reply backend/tracim_backend/lib/webdav/authentification.py]: done #1636

**Pull request merged**